cob_twist_controller/scripts/trajectory_marker_publisher_node.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python
+"""
+ * \file
+ *
+ * \note
+ *   Copyright (c) 2014 \n
+ *   Fraunhofer Institute for Manufacturing Engineering
+ *   and Automation (IPA) \n\n
+ *
+ *****************************************************************
+ *
+ * \note
+ *   Project name: care-o-bot
+ * \note
+ *   ROS stack name: cob_control
+ * \note
+ *   ROS package name: cob_twist_controller
+ *
+ * \author
+ *   Author: Marco Bezzon, email: Marco.Bezzon@ipa.fraunhofer.de
+ *
+ * \date Date of creation: July, 2015
+ *
+ * \brief
+ *   Simple Python node to publish a line strip to see the real trajectory and the desired one. 
+ *
+"""
+
+import time
+import roslib; roslib.load_manifest("interactive_markers")
+import rospy
+
+from visualization_msgs.msg import Marker
+from geometry_msgs.msg import Point
+from std_msgs.msg import ColorRGBA
+from interactive_markers.interactive_marker_server import *
+import tf
+
+
+def getMarker(root_frame, start_id, color):
+    # create a grey box marker
+    box_marker = Marker()
+    box_marker.type = Marker.LINE_STRIP
+    box_marker.scale.x = 0.03 # only this used for line_strip
+#    box_marker.scale.y = 0.2
+#    box_marker.scale.z = 0.1
+    box_marker.color.r = color.r
+    box_marker.color.g = color.g
+    box_marker.color.b = color.b
+    box_marker.color.a = color.a
+    
+    box_marker.pose.position.x = 0.0
+    box_marker.pose.position.y = 0.0
+    box_marker.pose.position.z = 0.0
+    box_marker.pose.orientation.x = 0.0
+    box_marker.pose.orientation.y = 0.0
+    box_marker.pose.orientation.z = 0.0
+    box_marker.pose.orientation.w = 1.0
+    
+    box_marker.ns = "basic_shapes"
+    box_marker.id = start_id
+    box_marker.action = Marker.ADD
+    box_marker.header.frame_id = root_frame
+    box_marker.header.stamp = rospy.Time.now()
+    
+    box_marker.lifetime = rospy.Duration()
+    return box_marker
+
+
+if __name__=="__main__":
+    rospy.init_node("simple_marker")
```

Comment:
I like this node!
We should extend it to be configurable and usable along any usage of the twist_controller...

 - Can we give the node a proper name?
 - Parameters should be retrieved from the parameter server using `get_param` as chain_tip_link, tracking_frame and root_frame are available there!
 - no `/` prefix in topics (so that they can be appended by the node's namespace)